application/testing/CMakeLists.txt

```diff
@@ -215,6 +215,31 @@ if(NOT F3D_MACOS_BUNDLE)
   f3d_test(NAME TestColorMapFile DATA dragon.vtu ARGS --colormap-file=magma.png --scalars --comp=1 DEFAULT_LIGHTS)
 endif()
 
+# Screenshot Interaction
+function(f3d_ss_test)
+  cmake_parse_arguments(F3D_SS_TEST "" "NAME;TEMPLATE;EXPECTED;DEPENDS" "ARGS" ${ARGN})
+  f3d_test(NAME TestScreenshot${F3D_SS_TEST_NAME} DATA suzanne.ply ARGS --screenshot-filename=${F3D_SS_TEST_TEMPLATE} --dry-run --interaction-test-play=${F3D_SOURCE_DIR}/testing/recordings/TestScreenshot.log NO_BASELINE DEPENDS TestClearScreenshots)
+  f3d_test(NAME TestScreenshot${F3D_SS_TEST_NAME}File DATA suzanne.ply ARGS --ref=${F3D_SS_TEST_EXPECTED} DEPENDS TestScreenshot${F3D_SS_TEST_NAME} TestScreenshot${F3D_SS_TEST_DEPENDS} NO_BASELINE LONG_TIMEOUT)
+endfunction()
+
+if(F3D_TESTING_ENABLE_LONG_TIMEOUT_TESTS)
```

Comment:
Here these screenshot actually long ? Since you are not using INTERACTION, you could put all these tests in normal testing.